Makefile

```diff
@@ -4,6 +4,10 @@ GIT ?= git
 CURL ?= curl
 SHELL := /bin/bash
 UNAME := $(shell uname | tr '[:upper:]' '[:lower:]')
+# Fetch latest stable release if 'ST2_VERSION' ENV var not set (ex: `2.7.1`)
+ST2_VERSION ?= $(shell curl --silent "https://api.github.com/repos/stackstorm/st2/releases/latest" | grep -Po '"tag_name": "v\K.*?(?=")')
+# Get today's date if 'BOX_VERSION' ENV var not set (ex: `20180507`)
+BOX_VERSION ?= $(shell date +%Y%m%d)
```

Comment:
Consider using UTC time?

```
BOX_VERSION ?= $(shell date -u +%Y%m%d)
```

Reply:
`ST2_VERSION` and `BOX_VERSION` should be derived from the environment variable `CIRCLE_TAG`. If the variable is not defined, then we won't be deploying to GitHub releases.. and the value of `ST2_VERSION` and `BOX_VERSION` can take the defaults set here. The caller can always override by setting the `ST2_VERSION` and `BOX_VERSION` environment variables before calling `make`. Right?

It's important that the value of `CIRCLE_TAG` matches the name of the `.ova` and `.box` files. Otherwise, the release page looks inconsistent:

![screen shot 2018-05-09 at 11 14 35 pm](https://user-images.githubusercontent.com/9972858/39854967-d4f82fd2-53de-11e8-827d-8012960c0de8.png)